testscript/doc.go

```diff
@@ -120,6 +120,35 @@ when testing.Short() is false.
 
 Additional conditions can be added by passing a function to Params.Condition.
 
+An example:
+
+		Condition: func(cond string) (bool, error) {
+			// Assume condition name and args are separated by colon (":")
+			args := strings.Split(cond, ":")
+			// Empty condition is already managed in testscript.run()
+			name := args[0]
+			switch name {
+			case "is_upper":
+				if len(args) < 2 {
+					return false, fmt.Errorf("syntax: [is_upper:word]")
+				}
+				return strings.ToUpper(args[1]) == args[1], nil
+			case "always_true":
+				return true, nil
```

Comment:
similar to my other comment, I'd like this to be slightly more realistic: in practice I don't think anyone would write an "always true" condition :)

How about one that's tied to an env var? For example, I've used "slow" or "long" env vars to tell `go test` to do more extensive testing, kinda like the opposite of `go test -short`. So you could have a `[long]` that checks `os.Getenv("TEST_LONG") == "true"`, for instance.